src/Atoms/ButtonsExercises/ButtonExercisesStyled.js

```diff
@@ -0,0 +1,25 @@
+import COLORS from '../../Styles/varsStyledComponents';
+import FONTS from '../../Styles/fontsStyledComponents';
+import styled from 'styled-components';
+
+const handleBgAndColor = (props) => {
+  if (props.endingButton) {
+    return `color: ${COLORS.grey800}; background: ${COLORS.grey100};`;
+  }
+  if (props.goBack) {
+    return `color: ${COLORS.primary801}; background: ${COLORS.primary201}; 
```

Comment:
prop shouldnt really care what does this button do. Its a dumb component, he should be told to display color, so for example the prop could be called color and woudl have 2 possible options: color="secondary", color="primary. If primary, display primary colors, etc.

Reply:
- names fixed